syntax = "proto3";

message setupProposal {
  string sender = 1;
  uint64 nonce = 2;
  uint64 minRefundBlock = 3;
  uint64 maxRefundBlock = 4;
}

message setupResponse {
  string recipient = 1;
  bytes lsigSignature = 2;
}

message Payment {
  uint64 cumulativeAmount = 1;
  bytes lsigSignature = 2;
}
